server/data/notificationsApiClient.test.ts

```diff
@@ -71,6 +72,7 @@ describe('GOV.UK Notify client', () => {
         {
           personalisation: {
             prison: confirmationDetails.prisonName,
+            'prison phone number': confirmationDetails.prisonPhoneNumber,
```

Comment:
I think we need to talk to the template creators about how to name fields 🙂

Reply:
Yeah, doesn't look great! Might be able to fix these ourselves as I think we're admins on the Notify account... 🤔?

Reply:
Camel case goes out the window in both of them templates on further inspection.. 
Do you think I should change this while I'm still working on this ticket?

Reply:
@hutcheonb-moj nah, use this ticket to get it working, we can go back and change the templates afterwards 🙂